circuits/src/zk_circuits/valid_match_settle/mod.rs

```diff
@@ -493,17 +585,6 @@ mod tests {
         assert!(!check_constraint_satisfaction::<SizedValidMatchSettle>(&witness, &statement));
     }
 
-    /// Test the case in which the `max_minus_min` field is incorrectly computed
-    #[test]
-    fn test_valid_match__incorrect_max_minus_min() {
```

Comment:
we removed this field from the witness (it was redundant) so this test is no longer necessary